rule_packages/c/Contracts1.json

```diff
@@ -0,0 +1,48 @@
+{
+  "CERT-C": {
+    "ENV30-C": {
+      "implementation_scope": {
+        "description": "None."
+      },
+      "properties": {
+        "obligation": "rule"
+      },
+      "queries": [
+        {
+          "description": "Return value of getenv and similar functions cannot be modified.",
```

Comment:
Suggested change -> 
"Modification of return values of getenv and similar functions results in undefined behaviour."